eng/mgmt/mgmtmetadata/azure-kusto_resource-manager.txt

```diff
@@ -3,13 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/azure-kusto/resource-manager/readme.md --csharp --version=v2 --reflect-api-versions --csharp-sdks-folder=C:\dev\sdks\azure-sdk-for-net\sdk
-Autorest CSharp Version: 2.3.91
-2022-02-15 22:23:13 UTC
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/azure-kusto/resource-manager/readme.md --csharp --version=2.0.4421 --reflect-api-versions --csharp-sdks-folder=C:\dev\azure-sdk-for-net\sdk
```

Comment:
The Kusto SDK is already moving to Track 2 https://www.nuget.org/packages/Azure.ResourceManager.Kusto/, so all new release should be Track 2 instead of Track 1.
Please let me know why still need Track 1 sdk, if only for Powershell, then you can update the Track 1 SDK and release a private signed SDK, don't publish to Nuget.